docs/pages/components/server.md

```diff
@@ -89,9 +89,13 @@ via `use()` will run only after the router has found a matching route. If no
 match is found, these handlers will never run. Takes a function, or an array
 of functions.
 
-## unfinishedRequests()
+## inflightRequests()
```

Comment:
there was another round of discussion where we felt `inflight` was more concise. cc @ktrott @rajatkumar